spec/services/session_service_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+describe SessionService do
```

Comment:
Is this test still being worked on?

Reply:
No .... Because it's done!  The tests should be in there now!